sites/next.skeleton.dev/src/examples/resources/cookbook/image-layouts/ExampleFeatured.astro

```diff
@@ -0,0 +1,28 @@
+<!-- Flex -->
+<section class="flex justify-center">
+	<!-- Grid -->
+	<div class="grid gap-4">
+		<!-- Featured -->
+		<div>
```

Comment:
Optional, but recommended: this `<div>` seems like a prime candidate to use `<header>`